src/pages/Committees.js

```diff
@@ -285,28 +340,33 @@ function Datasladden() {
 function DEMON() {
     return (
         <p>
-            Interested in music? DEMON (Swedish for 'CS and Media's orchestra club') is the joined committee 
+            Interested in music? DEMON (Swedish for "CS and Media's orchestra club") is the joined committee
             for music. They do not have a fixed repetition place, but can often be found at Hötorget. 
             There instruments can be freely borrowed, including a few guitars, a digital drum set and a digital 
-            piano. The head of committee is the ÄrkeDEMON (mail: arkedemon@datasektionen.se).
+            piano. The head of committee is the ÄrkeDEMON.
             <br/>
             To get in the committee, try asking people in META to get access to their Slack.
+            <br/>
+            Contact:
+            <ul>
+                <li>Email: arkedemon@datasektionen.se</li>
+            </ul>
         </p>
     )
 }
 
 function DESC() {
     return (
         <p>
-            DESC (Swedish for 'CS e-sport community') is the committee to look for if you are into gaming. They 
+            DESC (Swedish for "Computer Science chapter's e-sport community") is the committee to look for if you are into gaming. They
```

Comment:
```suggestion
            DESC (standing for "Datasektionens E-Sports Community") is the committee to look for if you are into gaming. They
```
This is hardly Swedish, Swenglish at best